README.md

```diff
@@ -73,3 +81,103 @@ optional arguments:
                         associated matrix conditions, a total of Nx2 builds
                         will be identified. (default: 2)
```

Comment:
This help text is out of sync with other changes in this PR. The code in `cli.py` now defines this as `(default: 2 if no matrix conditions)`. Similarly for `--matrix-max-n-major-versions`.